etc/config/c++.amazon.properties

```diff
@@ -277,7 +277,7 @@ compiler.g142assert.semver=14.2 (assertions)
 
 ################################
 # Clang for x86
-group.clang.compilers=&clangx86assert:clang30:clang31:clang32:clang33:clang341:clang350:clang351:clang352:clang37x:clang36x:clang371:clang380:clang381:clang390:clang391:clang400:clang401:clang500:clang501:clang502:clang600:clang601:clang700:clang701:clang710:clang800:clang801:clang900:clang901:clang1000:clang1001:clang1100:clang1101:clang1200:clang1201:clang1300:clang1301:clang1400:clang1500:clang1600:clang1701:clang1810:clang1910
+group.clang.compilers=&clangx86assert:clang30:clang31:clang32:clang33:clang341:clang350:clang351:clang352:clang37x:clang36x:clang371:clang380:clang381:clang390:clang391:clang400:clang401:clang500:clang501:clang502:clang600:clang601:clang700:clang701:clang710:clang800:clang801:clang900:clang901:clang1000:clang1001:clang1100:clang1101:clang1200:clang1201:clang1300:clang1301:clang1400:clang1500:clang1600:clang1701:clang1810:clang1810_clad180:clang1910
```

Comment:
Why didn't this go to `group.clangx86trunk.compilers`, like all other Clang variants that are not upstream releases?

Reply:
`clad` is not a separate version, it's just a normal clang + the clad plugin, it's version specific. We might add a new group when more versions appear.

Reply:
I'm definitely looking forward to move it somewhere else, because the list of just Clang versions is long enough already.